qt_ui/windows/mission/flight/settings/QFlightSlotEditor.py

```diff
@@ -131,6 +136,8 @@ def on_player_toggled(self, checked: bool) -> None:
         pilot.player = checked
         self.player_toggled.emit()
 
+        self.pilots_changed.emit()
+
     def on_pilot_changed(self, index: int) -> None:
```

Comment:
`on_pilot_changed` should emit `pilots_changed` in its `finally` block, otherwise the start-type isn't updated if you have a single client pilot which you switch to a non-client pilot...

Reply:
Done 👍